frontend/src/pages/ExperimentList.tsx

```diff
@@ -181,6 +182,8 @@ class ExperimentList extends Page<{}, ExperimentListState> {
         request.pageSize,
         request.sortBy,
         request.filter,
+        this.props.namespace ? 'NAMESPACE' : undefined,
```

Comment:
Just curious, do we have any constant variable that represents the resource reference type?

Reply:
There are some problems with current swagger codegen version for frontend. The generated enum cannot pass typechecking here. (tech details...)

But don't worry about the constant here, the function actually accepts only reasonable constants (enforced by typescript typing system): https://github.com/Bobgy/pipelines/blob/10498bf1ad2cd466da59cdb5d58343fa71138ab4/frontend/src/apis/experiment/api.ts#L435-L441